phoenixRest/models/core/event_type.py

```diff
@@ -0,0 +1,39 @@
+from sqlalchemy import (
+    Column,
+    DateTime,
+    ForeignKey,
+    Integer,
+    UniqueConstraint,
+    Text,
+    Integer,
+    Boolean,
+    Enum,
+    Table
+)
+from sqlalchemy.dialects.postgresql import UUID
+
+from sqlalchemy.orm import relationship
+from sqlalchemy import and_
+
+from phoenixRest.models import Base
+
+import logging
+log = logging.getLogger(__name__)
+
+import uuid
+
+class EventType(Base):
+    __tablename__ = "event_type"
+    
+    uuid = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4, unique=True, nullable=False)
```

Comment:
I think we agreed to use a string instead of uuid here, since people will need to know/remember it when configuring clients?

Reply:
so like a `slug` instead of uuid? (which is a string which we enforce to be lowercase letters and `-_` or something?